examples/nextjs-with-typescript/package.json

```diff
@@ -7,6 +7,7 @@
     "@emotion/react": "latest",
     "@emotion/styled": "latest",
     "@emotion/server": "latest",
+    "@material-ui/icons": "latest",
```

Comment:
This package is unused in the example. If you expand the example locally then you have to add the package manually. Adding packages that are unused is confusing as well.